### ISSUE TITLE
[v1.10.0] Generate an `unreleased` section is not possible

### ISSUE THREAD

**Issue by llaville** [miniscruff/changie#407]
**Description**
Changie v1.10.0 is not able to generate an `unreleased` section like in [Keep a Changelog 1.0.0](https://keepachangelog.com/en/1.0.0/)

**Reproduction Steps**
1. Run `docker run --rm --user $(id -u):$(id -g) --mount type=bind,source=$PWD,target=/src -w /src  -it ghcr.io/miniscruff/changie:v1.10.0 batch --keep unreleased`
1. See error

**What happened**
Display following error with Changie help screen `Error: part string is not a supported version or version increment`

**Expected behavior**
Be able to generate a `.changes/unreleased.md` file 

**Additional context**
When I run `docker run --rm --user $(id -u):$(id -g) --mount type=bind,source=$PWD,target=/src -w /src  -it ghcr.io/miniscruff/changie:v1.10.0 merge` with an `.changes/unreleased.md` file (created by hand) with valid content (as a prerelease version), `CHANGELOG.md` did not contains its value and no error messages are displayed.

**Comment by miniscruff** [comment]
Unreleased is not a valid semver string and thus does not work as a version. It is true that keep a changelog suggests an unreleased section but this doesn't quite work in Changie and the best substitute would be to look at the files in the unreleased folder. You could generate prereleases but it would not be possible to generate an unreleased version. Something like `vx.y.z-rc1` would work if it matches your workflow. This sort of relates back to versions being immutable, which I guess contradicts keep a changelog a little here, and thus need to be versioned properly and released only once.

tl;dr this section of the guide ( https://keepachangelog.com/en/1.0.0/#effort ) is what Changie aims to make easier and does not really function in Changie as it stands now or probably ever. I would point to GitLab's blog on why this method is not ideal at scale anyway https://about.gitlab.com/blog/2018/07/03/solving-gitlabs-changelog-conflict-crisis/.

**Comment by llaville** [comment]
OK I've adapted my CI (GitHub Workflow) to generate a snapshot release on fly without the need to commit a `.changes/<version>-snapshot.md` file. 
That will simulate for me the equivalent to https://keepachangelog.com/en/1.0.0/#effort
